core/common-util/test/ObjectUtil.test.ts

```
@@ -1,6 +1,27 @@
 import assert from 'node:assert';
 import { ObjectUtils } from '..';
 
+export function InitTypeQualifier() {
+  return function(_target: any, _propertyKey?: PropertyKey, _parameterIndex?: number) {
+    console.log(_target, _propertyKey, _parameterIndex);
+    // ...
+  };
+}
```

Biome

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)